drivers/src/cpu.rs

```diff
@@ -79,6 +81,16 @@ fn isa_string_has_extension(isa_string: &str, extension: &str) -> bool {
     isa_string.split('_').any(|f| f == extension)
 }
 
+fn isa_string_has_base_extension(isa_string: &str, extension: char) -> bool {
+    isa_string
+        .split('_')
+        .next()
+        .expect("ISA string too short")
```

Comment:
Should we panic if no features are detected?

Reply:
is that even possible? next() will just return rv64i....

there is always at least one, isn't there?

Reply:
Sure, it's nearly certain that we'll have at least one feature, but what I meant is that we probably don't need to go out of our way to panic (when we can just easily mark the feature as not detected).